provider/spark.go

```diff
@@ -37,8 +38,9 @@ import (
 type SparkExecutorType string
 
 const (
-	EMR        SparkExecutorType = "EMR"
-	Databricks                   = "DATABRICKS"
+	EMR          SparkExecutorType = "EMR"
+	Databricks                     = "DATABRICKS"
+	SparkGeneric                   = "SPARK"
 )
```

Comment:
My Go language server complains that only the first constant in the list is typed. I'm assuming that all three of these entries should be a string of type `SparkExecutorType`, right?